setup/sql/5_4_hotfixes.php

```diff
@@ -1248,5 +1248,38 @@
     $ilDB->execute($updateStatement, [$row['certificate_content'], $row['id']]);
 }
 ?>
+<#83>
+<?php
+$checked_files = [];
+$defaulf_bg_img_path = '/certificates/default/background.jpg';
+$checked_files[$defaulf_bg_img_path] = file_exists(CLIENT_DATA_DIR . $defaulf_bg_img_path);
+
+$res = $ilDB->query('SELECT id, certificate_content_bu FROM il_cert_user_cert WHERE background_image_path = "[BACKGROUND_IMAGE]"');
+$updateStatement = $ilDB->prepareManip("UPDATE il_cert_user_cert SET background_image_path = ? WHERE id = ?", ['text', 'integer']);
```

Comment:
Is it sufficient to modify the achieved user certificates only? Don't we need to fix the certificate templates as well?

Reply:
As I can see so far, the problem seems to only concern the user certificates. The templates have correct entries at the `background_image_path` column, at least after the latest hotfix step.
I will investigate this some more. The whole problem is very strange and until now I could not figure out how it was possible in any case, that this happened.